distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestCatalog.java

```diff
@@ -136,6 +140,76 @@ public void testCswIngest() {
                 hasXPath("//TransactionResponse/InsertResult/BriefRecord/BoundingBox"));
     }
 
+    @Test
+    public void testFilterPlugin() {
+        try {
+            // Ingest the metacard
+            String id1 = ingestXmlFromResource("/metacard1.xml");
+
+            // Test without filtering
+            ValidatableResponse response = executeOpenSearch("xml", "q=*");
+            response.body(hasXPath("/metacards/metacard[@id='" + id1 + "']"));
```

Comment:
Even at one instance alone I think String.format is more readable (but i do string concat all the time, so call me a hypocrite).. but also the fact that we use this identical string more than once means that we could define it along with id1, and then reduce code the other times we use it (168 & 174).

So maybe a `String metacardXPath = "/metacards/metacard[@id='%s']";`
So that we can just do `response.body(hasXPath(String.format(metacardXPath, id1)));`

I suppose it's still long, though having it defined only once makes it a little more maintainable. If java had string literal interpolation built in it would all be dandy... *sigh*